faq/account.mdx

```diff
@@ -0,0 +1,160 @@
+---
+meta:
+  title: Account
+  description: Get answers to your most common account-related questions.
+content:
+  h1: Account
+hero: assets/general.webp
+---
+
+<Concept>
+
+## My account is locked. What do I do?
+
+If your account is locked, contact our support team to learn the reason for the blockage and discover how to regain access.
```

Comment:
Maybe add an email address or info of how to contact them? I guess if the account is locked they can't log in to the console to open a ticket?